src/main/java/uk/gov/hmcts/ethos/replacement/docmosis/helpers/TseHelper.java

```diff
@@ -333,4 +339,10 @@ public static String formatRule92(String copyToOtherPartyYesOrNo, String copyToO
                         : ""
         );
     }
+
+    private static void ensureSelectedApplicationExists(GenericTseApplicationType genericTseApplicationType) {
+        if (genericTseApplicationType == null) {
+            throw new IllegalStateException("Selected application is null");
+        }
```

Comment:
looks like you can throw maybe here instead of returning null

https://github.com/hmcts/et-ccd-callbacks/blob/ee3af2b7dfec933ce5951b25cd5ea7573e3c2da0/src/main/java/uk/gov/hmcts/ethos/replacement/docmosis/helpers/TseHelper.java#L210